packages/cli/src/commands/setup/deploy/deploy.js

```diff
@@ -160,5 +160,6 @@ export const handler = async ({ provider, force }) => {
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
    console.error(c.error(e.message))
```